src/components/results/dynamicsimulation/dynamic-simulation-result-tab.js

```diff
@@ -52,28 +56,33 @@ const DynamicSimulationResultTab = ({ studyUuid, nodeUuid }) => {
                     />
                 </Tabs>
             </Box>
-            <Box>
+            <Box sx={styles.resultContainer}>
                 <TabPanelLazy
                     key={TAB_INDEX_TIME_SERIES}
                     selected={tabIndex === TAB_INDEX_TIME_SERIES}
                 >
-                    {dynamicSimulationResultPresent && (
-                        <DynamicSimulationResultTimeSeries
-                            studyUuid={studyUuid}
-                            nodeUuid={nodeUuid}
-                        />
-                    )}
+                    <DynamicSimulationResultTimeSeries
+                        studyUuid={studyUuid}
+                        nodeUuid={nodeUuid}
+                    />
+                </TabPanelLazy>
+                <TabPanelLazy
+                    key={TAB_INDEX_TIME_LINE}
```

Comment:
Useless key prop

Reply:
YES! We do not render a list, neither in a particular case which needs dynamic keys!
REMOVED!